### ISSUE TITLE
Readme: differentiate from Serverspec

### ISSUE THREAD

**Issue by joshjordan** [inspec/inspec#229]
Kudos for giving a nod to Serverspec.

It would be really helpful to explain why this is different from serverspec and why someone might choose to use inspec instead of serverspec.

**Comment by chris-rock** [comment]
@joshjordan Does the [blog post](https://www.chef.io/blog/2015/11/04/the-road-to-inspec/) covers what you need?

**Comment by bradenwright** [comment]
Blog covered it for me.  I could see it being nice to have it in a number of bullet points for quick consumption.  But the blog was a good read, and I think covered my questions on the topic nicely.

**Comment by joshjordan** [comment]
Its a great start. It definitely covers motivation, but I'm not sure if it is saying that inspec is meant to supercede Serverspec, or if there are use cases for both.

In any case, if the blog is the thing people should read, we should get a link to that in the readme. I would prefer something much more succinct, though. I don't think people are going to want to read the entire blog post to help them make that choice.

**Comment by mvdkleijn** [comment]
I'd also love to know how InSpec is positioned against http://infrataster.net/ which basically seems to have the same goal...

**Comment by ryancurrah** [comment]
+1 would like to know if inspec supersedes sever spec.

**Comment by jsirex** [comment]
Even after reading all the blogs, examples I can't understand "why inspec"?
Want smart resource, like sshd_config - define it with serverspec. Why not to improve existing software? Now community has divided between serverspec and inspec?
What can I do easily with inspec which can't with serverspec?

**Comment by benpatterson** [comment]
I also don't understand why inspec is a competing framework, as opposed to an add-on or set of assertions that go on top of serverspec. the blog post did not clear that up for me when I read through it.

**Comment by Sinjo** [comment]
So I ended up finding my answers in [this podcast](http://foodfightshow.org/2016/02/inspec.html).

I took a tonne of notes, but they're in a totally raw form right now. If it'd be helpful, and something maintainers would like, I'll happily tidy them up and PR them as a doc to this repo - perhaps linked to from the `README`.

Let me know either way. It's most likely I'll have time to do this in February.
